xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-data/pom.xml

```diff
@@ -161,6 +161,12 @@
               <artifactId>xwiki-platform-ratings-api</artifactId>
               <version>${project.version}</version>
             </dependency>
+            <!-- Used to process the document analysis tasks (e.g., backlinks indexing). -->
+            <dependency>
```

Comment:
Not really needed here since it's already a dependency of the plugin.

Reply:
Indeed, removing it, thanks.